website/docs/cloud-docs/users-teams-organizations/single-sign-on/linking-user-account.mdx

```diff
@@ -18,7 +18,10 @@ The first time you use SSO to log in to an organization, Terraform Cloud links t
 When Terraform Cloud does not recognize the email address associated with your identity provider, it asks if you want to create a new user account with that email address. You can choose one of the following actions:
 
 - **Create a new account:** Terraform Cloud automatically links your SSO identity to this new account after creation.
-- **Link to an existing account:**  Click **Link SSO identity to a different account** and sign in with a Terraform Cloud account. Terraform Cloud automatically links your SSO identity to this account when you log in.
+- **Link to an existing account:** Click **Link SSO identity to a different account** and sign in with a Terraform Cloud account.
+  - For **non-unified accounts**: If your account is not HCP-linked or you use username/password to sign in then you have a non-unified account. Sign in with a Terraform Cloud account username/password and Terraform Cloud automatically links your SSO identity to this account.
+  - For **HCP-linked accounts**: Click the "Continue with HCP account" button. You will be prompted for your HCP credentials to sign into your Terraform Cloud account. Terraform Cloud automatically links your SSO identity to the HCP-linked account.
```

Comment:
```suggestion
- **Link to an existing account:** Click **Link SSO identity to a different account** and sign in with one of the following account types.
     - **HashiCorp Cloud Platform (HCP) account**: Click **Continue with HCP account** and use your HCP credentials to sign in to Terraform Cloud. Terraform Cloud automatically links your SSO identity to that HCP account. Refer to [Linked HCP and Terraform Cloud Accounts](/cloud-docs/users-teams-organizations/users#linked-hcp-and-terraform-cloud-accounts) for more details.
  - **Terraform Cloud account**: Sign in with your Terraform Cloud username and password. Terraform Cloud automatically links your SSO identity to that account.
```

Reply:
hm I would consider a HCP-linked account a TFC account. We don't have a term for TFC accounts that have not been linked to a HCP account yet ( which I referred to as `non-unified accounts`). But I do see that this could be confusing to a user that does not know if their TFC account is linked to a HCP account or not.

Reply:
To the user the perceived difference is how they authenticate their TFC account. HCP-linked accounts use HCP auth and "non-unified accounts" use username/pwd auth. Perhaps I can rewrite these directions and frame it as "use the sign-in method that you normal use to sign into your TFC account: either "Continue with HCP account" for HCP-linked TFC accounts or username/password for all other TFC accounts

Reply:
hmm I see your point! I mean a "non-unified" account is just a TFC account though, right? (lmk if I'm missing something - I really could be!)

BUT I see your point about the HCP Linking. I adjusted the recommendation to be "Linked HashiCorp Cloud Platform account". Do we like that better?

Reply:
yep, a non-unified account is just a TFC account that has not been linked to a HCP account yet. It's very much a made up term and I'm not sure if it would make sense to the average TFC user but it is the best term I came up with when I asked others what terminology they'd use. I really like your suggestion! 

One minor tweak:`links your SSO identity to that HCP account` → `links your SSO identity to that HCP-linked account`  since we aren't actually doing any linking to the HCP account but to the TFC account that has a unified identity/is linked to a HCP account